### ISSUE TITLE
19.5 Species list

### ISSUE THREAD

**Issue by gubuntu** [kartoza/WBR-SEMP#60]
Provide a list of important species occurring within the proposed biosphere reserve, including common names, wherever possible

ref biosphere_reserve_nomination_form_2013_en.pdf

**Comment by zacharlie** [comment]
The form request is `Provide a list of important species occurring within the proposed biosphere reserve, including common names, wherever
possible.`, however additonal information may be required for #54

Spatial data and IUCN Red List Categories and Criteria (which are used by SANBI) are available from the https://www.iucnredlist.org but the license is restricted for commercial activity.

The data is available from http://redlist.sanbi.org/redcat.php but it is not in a data format that may be easily cross referenced.

This is also only for threatened specices and does not give an indication of other endemic or important species.

Species of importance data for birding is detailed from the important bird areas web data. The original data links:

- http://www.birdlife.org.za/conservation/important-bird-areas/iba-directory/item/149-sa008-nyl-river-floodplain
- http://www.birdlife.org.za/conservation/important-bird-areas/iba-directory/item/148-sa007-waterberg-system
- http://www.birdlife.org.za/conservation/important-bird-areas/iba-directory/item/150-sa009-northern-turf-thornveld
- http://www.birdlife.org.za/conservation/important-bird-areas/iba-directory/item/164-sa023-pilansberg-national-park

were not available and had to be manually sourced, which resulted in the following pages:

- https://www.birdlife.org.za/iba-directory/nyl-river-floodplain/
- https://www.birdlife.org.za/iba-directory/waterberg-system/
- https://www.birdlife.org.za/iba-directory/northern-turf-thornveld/
- https://www.birdlife.org.za/iba-directory/pilansberg-national-park/

Whilst these pages offer some information on species they do not supply full citations and rely on a "Further Reading" section.

Species of importance must be manually extracted from each page and added into a suitable data format.

Additional information on important species is listed on the UNESCO website https://en.unesco.org/biosphere/africa/waterberg, however this does not seem to give any indication of wher that data was sourced.

A copy of the [Waterberg District Bioregional Plan](https://bgis.sanbi.org/Document/Download?id=2181) Draft 20160201 was sourced in PDF format which contains details on particular species of concern, however the information does not contain specific citation and the document references do not make any explicit mention of species data.

Additional Biome information data is avialable from NBA2018_Terrestrial_ThreatStatus_ProtectionLevel, however it is not species specific to a specific region within the area of interest and simply contains vegetation/ biome types which usually support a range of species, e.g. Springbokvlakte Thornveld. It does, however, give an indication of endemic biome type.

Species of critical importance may be derived from the NATIONAL ENVIRONMENTAL MANAGEMENT: BIODIVERSITY ACT and a 2007 Gazetted list of CRITICALLY ENDANGERED, ENDANGERED, VULNERABLE AND PROTECTED SPECIES is available from www.speciesstatus.sanbi.org/pdf/NEMBAToPslist23Feb.pdf

A complete list of species occurences was sourced for South Africa from the [Global Biodiversity Information Facility](https://en.wikipedia.org/wiki/Global_Biodiversity_Information_Facility) data base.

This occurence data does not provide information on the importance of the species.

The resulting list provides 10390 distinct species values, with 9260 passing some elementary integrity checks and 1130 records indicating invalid values or conventions. No common names are available from this data.

The raw data list is available from [species_list.csv](https://github.com/kartoza/WBR-SEMP/blob/master/species_list/species_list.csv)

This is a comprehensive list of species observations, and does not directly serve the purposes of this species list but may serve to provide additional analysis and reference functions. As the sources listed above provide little data which is overtly available to support their claims, it may prove to be a useful resource. It has been provisionally commited to the git repository for the sake of posterity, however a refined list of greatly reduced scope will replace it and is currently in development.

The citation data for these GBIF observations is as follows:

```
Citation: GBIF.org (08 October 2020) GBIF Occurrence Download https://doi.org/10.15468/dl.zyvv6f
License: Unspecified


INCLUDES RECORDS FROM 1,425 DATASETS
```

The resulting records for the entire search (~28000000), were processed into spatial positions. The full processing outline is available from my personal repository under [gbif_simple_sql.md](https://github.com/zacharlie/effective-dollop/blob/main/sql/gbif_simple_sql.md).

It is possible that additional analysis may be performed by cross referencing these data sources or by running spatial analysis on occurence data for particular species of interest where relevant.

Species data will need to be prioritised and consolidated appropriately. A [species list template](https://bgis.sanbi.org/Document/Download?id=1582) is available from SANBI, however it does not seem very well suited to the data inputs available and may require a significant amount of additional work to format avialable data into this format.

**Comment by zacharlie** [comment]
Localised resources were taken from relevant listed sources and captured into a suitable data format, and local occurrences of a species were obtained using the derived gbif data and cross referencing it against lists of species convcern which occur at a national level, such as NEMBA.

An additional resource was obtained from the SANBI [POSA](http://newposa.sanbi.org/) which supplied a [sensitive species list](http://biodiversityadvisor.sanbi.org/wp-content/uploads/2015/08/sensitive_species_list_2010.xlsx) for flora spicies of national significance in South Africa.

A 2011 [LIST OF ECOSYSTEMS THAT ARE THREATENED AND IN NEED OF PROTECTION](http://biodiversityadvisor.sanbi.org/wp-content/uploads/2012/10/20111209-National-Gazette-No-34809-of-09-December-2011-Volume-558.pdf) as outlined by the NATIONAL ENVIRONMENTAL MANAGEMENT: BIODIVERSITY ACT was also obained, however it was not in a suitable format for inclusion in this report and will require additional resource allocation to verify the special concern species for the study area.

**Comment by zacharlie** [comment]
Data available from #80 

Species status was not always well defined in the source data and a number of synonyms were used, however these introduce some ambiguity as to the exact status of threat of a species or it's protections. As such, the status details were summarised into the following categories, in descending priority.

ENDANGERED - Species listed as endangered by source or listed on the Red List (according to UNESCO) or NEMBA protected species list data.
THREATENED - Species listed as threatened.
SENSITIVE - Species on the POSA sensitive species list (flora only).
VULNERABLE - Species listed as threatened.
TRIGGER - Species with other categorisations. These are trigger species for conservation action and may be regionally threatened, endemic, protected or otherwise significant (such as rare, biome restricted or congregatory species) within the context of the current study area.

As a result, this list does not comprehensively or accurately outline the current threat level for any of the species listed and does not properly distinguish between threatened, critically endangered or recovering species. As a general rule, endangered species are of national importance or populations are at critical levels, threatened species are internationally threatened, and vulnerable are of regional importance or threatened locally. This is a general guideline and is NOT a rule, as inaccuracies are expected to be present within this dataset.

Where duplicated records were identified, a single result was retained with the highest priority status assigned. Where the category or family of a species was not defined, it was surmised from the available resources and consolidated to match the categories outlined in the NEMBA list.

The complete data has been included in spread sheet format. Note that significant manual error correction, such as grammatical or clerical errors, in any of the source done was done in an ad hoc manner and was not comprehensive. Note as well that certain species are elusive, and lack of an occurence within the obtained data does not necessarily indicate that an area of interest or biome is not of significance to contributions for the conservation efforts of the species.

The result of this assessment yielded a total number of 188 distinct species of significance within the study area exent.

## Summary

| STATUS     | No of Species |
|------------|---------------|
| ENDANGERED | 24            |
| THREATENED | 23            |
| SENSITIVE  | 11            |
| VULNERABLE | 37            |
| TRIGGER    | 93            |

| Category     | No of Species |
|--------------|---------------|
| AMPHIBIA     | 2             |
| AVES         | 89            |
| FLORA        | 41            |
| INVERTEBRATA | 2             |
| MAMMALIA     | 33            |
| PISCES       | 1             |
| REPTILIA     | 20            |

## Species List

| Status     | Category     | Scientific Name               | Common Name                        |
|------------|--------------|-------------------------------|------------------------------------|
| TRIGGER    | FLORA        | acacia                        |                                    |
| VULNERABLE | MAMMALIA     | acinonyx jubatus              | cheetah                            |
| TRIGGER    | REPTILIA     | afroedura langi               | lowveld flat gecko                 |
| TRIGGER    | AVES         | alcedo semitorquata           | half-collared kingfisher           |
| TRIGGER    | REPTILIA     | amblyodipsas concolor         | natal purple-glossed snake         |
| TRIGGER    | REPTILIA     | amblyodipsas concolor         | natal purple-glossed snake         |
| TRIGGER    | REPTILIA     | amblyodipsas ventrimaculata   | kalahari purple-glossed snake      |
| VULNERABLE | MAMMALIA     | amblysomus julianae           | juliana’s golden mole              |
| SENSITIVE  | FLORA        | ansellia africana             |                                    |
| THREATENED | AVES         | anthropoides paradiseus       | blue crane                         |
| TRIGGER    | AVES         | anthus lineiventris           | striped pipit                      |
| VULNERABLE | AVES         | aquila rapax                  | tawny eagle                        |
| TRIGGER    | AVES         | aquila spilogaster            | african hawk eagle                 |
| TRIGGER    | AVES         | aquila verreauxii             | verreauxs’ eagle                   |
| TRIGGER    | AVES         | ardeola ralloides             | squacco heron                      |
| THREATENED | AVES         | ardeotis kori                 | kori bustard                       |
| TRIGGER    | REPTILIA     | aspidelaps scutatus           | shield-nosed snake                 |
| TRIGGER    | MAMMALIA     | atelerix frontalis            | southern african hedgehog          |
| TRIGGER    | REPTILIA     | atractaspis duerdeni          | duerden’s burrowing asp            |
| ENDANGERED | AVES         | botaurus stellaris            | bittern                            |
| VULNERABLE | FLORA        | bowiea volubilis              |                                    |
| VULNERABLE | FLORA        | brachycorythis conica         |                                    |
| ENDANGERED | FLORA        | brachystelma gerrardii        |                                    |
| TRIGGER    | AVES         | bradornis mariquensis         | marico flycatcher                  |
| TRIGGER    | REPTILIA     | bradypodion taeniabronchum    | smith’s dwarf chameleon            |
| TRIGGER    | AVES         | bubo capensis                 | cape eagle-owl                     |
| THREATENED | AVES         | bucorvus leadbeateri          | southern ground-hornbill           |
| TRIGGER    | AVES         | buteo rufofuscus              | jackal buzzard                     |
| TRIGGER    | AVES         | c. pectoralis                 | black-chested snake eagle          |
| TRIGGER    | AVES         | calamonastes fasciolatus      | barred wren-warbler                |
| TRIGGER    | AVES         | campicoloides bifasciata      | buff-streaked chat                 |
| THREATENED | MAMMALIA     | ceratotherium simum           | white rhinoceros                   |
| VULNERABLE | FLORA        | ceropegia stentiae            |                                    |
| VULNERABLE | AVES         | ciconia nigra                 | black stork                        |
| TRIGGER    | AVES         | cinnyris talatala             | white-bellied sunbird              |
| TRIGGER    | AVES         | circaetus cinereus            | brown snake eagle                  |
| TRIGGER    | AVES         | circus ranivorus              | african marsh harrier              |
| SENSITIVE  | FLORA        | clivia miniata                |                                    |
| ENDANGERED | MAMMALIA     | cloeotis percivali            | short-eared trident bat            |
| TRIGGER    | MAMMALIA     | connochaetes gnou             | black wildebeest                   |
| THREATENED | AVES         | coracias garrulus             | european roller                    |
| VULNERABLE | FLORA        | corchorus psammophilus        |                                    |
| TRIGGER    | REPTILIA     | cordylus breyeri              | waterberg girdled lizard           |
| TRIGGER    | AVES         | cossypha humeralis            | white-throated robin-chat          |
| TRIGGER    | AVES         | crex crex                     | corn crake                         |
| VULNERABLE | MAMMALIA     | cricetomys gambianus          | giant rat                          |
| ENDANGERED | REPTILIA     | crocodylus niloticus          | nile crocodile                     |
| TRIGGER    | MAMMALIA     | crocuta crocuta               | spotted hyaena                     |
| SENSITIVE  | FLORA        | cryptocarya transvaalensis    |                                    |
| VULNERABLE | FLORA        | cucumis humifructus           |                                    |
| VULNERABLE | FLORA        | cullen holubii                |                                    |
| SENSITIVE  | FLORA        | curtisia dentata              |                                    |
| SENSITIVE  | FLORA        | cyphostemma hardyi            |                                    |
| ENDANGERED | MAMMALIA     | damaliscus lunatus            | tsessebe                           |
| ENDANGERED | FLORA        | delosperma macellum           |                                    |
| TRIGGER    | AVES         | dendrocygna viduata           | white-face whistling duck          |
| ENDANGERED | MAMMALIA     | diceros bicornis              | black rhinoceros                   |
| TRIGGER    | FLORA        | digitaria eriantha            |                                    |
| SENSITIVE  | FLORA        | dioscorea sylvatica           |                                    |
| SENSITIVE  | FLORA        | drimia sanguinea              |                                    |
| TRIGGER    | AVES         | egretta alba                  | great egret                        |
| TRIGGER    | AVES         | egretta garzetta              | little egret                       |
| TRIGGER    | AVES         | egretta intermedia            | yellow-billed egret                |
| TRIGGER    | AVES         | egretta vinaceigula           | slaty egret                        |
| SENSITIVE  | FLORA        | elaeodendron transvaalense    |                                    |
| VULNERABLE | FLORA        | elytrophorus globularis       |                                    |
| ENDANGERED | FLORA        | encephalartos eugene-maraisii | waterberg cycad                    |
| TRIGGER    | FLORA        | encephalartos transvenosus    | modjadje cycad                     |
| ENDANGERED | AVES         | ephippiorhynchus senegalensis | saddle-billed stork                |
| TRIGGER    | FLORA        | eragrostis pallens            |                                    |
| TRIGGER    | AVES         | eremomela usticollis          | burnt-necked eremomela             |
| THREATENED | INVERTEBRATA | eriksonia acraeina            | butterfly species                  |
| TRIGGER    | FLORA        | erythrophysa transvaalensis   |                                    |
| TRIGGER    | AVES         | erythropygia paena            | kalahari scrub robin               |
| TRIGGER    | AVES         | erythropygia paena            | kalahari scrub robin               |
| TRIGGER    | AVES         | estrilda erythronotos         | black-faced waxbill                |
| VULNERABLE | FLORA        | eulophia coddii               |                                    |
| ENDANGERED | FLORA        | euphorbia clivicola           |                                    |
| SENSITIVE  | FLORA        | euphorbia groenewaldii        |                                    |
| VULNERABLE | AVES         | eupodotis caerulescens        | blue korhaan                       |
| TRIGGER    | AVES         | eupodotis senegalensis        | white-bellied korhaan              |
| TRIGGER    | AVES         | eurocephalus anguitimens      | southern white-crowned shrike      |
| TRIGGER    | AVES         | falco biarmicus               | lanner falcon                      |
| VULNERABLE | AVES         | falco naumanni                | lesser kestrel                     |
| VULNERABLE | AVES         | falco peregrinus              | peregrine falcon                   |
| THREATENED | MAMMALIA     | felis lybica                  | african wildcat                    |
| TRIGGER    | MAMMALIA     | felis nigripes                | black-footed cat                   |
| TRIGGER    | AVES         | gallinago nigripennis         | african snipe                      |
| TRIGGER    | AVES         | gallinula angulata            | lesser moorhen                     |
| ENDANGERED | FLORA        | gasteria disticha             |                                    |
| VULNERABLE | AVES         | geronticus calvus             | bald ibis                          |
| TRIGGER    | AVES         | glareola nordmanni            | black-winged pratincole            |
| ENDANGERED | AVES         | gorsachius leuconotus         | white-backed night heron           |
| ENDANGERED | AVES         | gyps africanus                | white-backed vulture               |
| ENDANGERED | AVES         | gyps coprotheres              | cape vulture                       |
| TRIGGER    | INVERTEBRATA | haliotis midae                | south african abalone              |
| TRIGGER    | FLORA        | harpagophytum procumbens      | devil’s claw                       |
| SENSITIVE  | FLORA        | haworthia lockwoodii          |                                    |
| TRIGGER    | AVES         | hieraaetus wahlbergi          | wahlberg’s eagle                   |
| VULNERABLE | MAMMALIA     | hippopotamus amphibious       | hippopotamus                       |
| THREATENED | MAMMALIA     | hippotragus equinus           | roan antelope                      |
| VULNERABLE | MAMMALIA     | hippotragus niger             | sable antelope                     |
| THREATENED | REPTILIA     | homopholis mulleri            | muller’s velvet gecko              |
| TRIGGER    | AVES         | ixobrychus minutus            | little bittern                     |
| TRIGGER    | AVES         | ixobrychus sturmii            | dwarf bittern                      |
| VULNERABLE | FLORA        | jamesbrittenia bergae         |                                    |
| TRIGGER    | AVES         | kurrichane thrush             | turdus libonyanus                  |
| VULNERABLE | MAMMALIA     | laephotis botswanae           | botswana long-eared bat            |
| TRIGGER    | AVES         | lamprotornis australis        | burchell’s starling                |
| TRIGGER    | AVES         | laniarius atrococcineus       | crimson-breasted shrike            |
| VULNERABLE | FLORA        | ledebouria atrobrunnea        |                                    |
| TRIGGER    | MAMMALIA     | leptailurus serval            | serval                             |
| TRIGGER    | AVES         | leptoptilos crumeniferus      | marabou stork                      |
| TRIGGER    | AVES         | lophotis ruficrista           | red-crested korhaan                |
| VULNERABLE | MAMMALIA     | loxodonta africana            | african elephant                   |
| ENDANGERED | MAMMALIA     | lycaon pictus                 | african wild dog                   |
| SENSITIVE  | FLORA        | lydenburgia cassinoides       |                                    |
| TRIGGER    | REPTILIA     | lygodactylus waterbergensis   | waterberg dwarf gecko              |
| VULNERABLE | MAMMALIA     | manis temminckii              | pangolin                           |
| VULNERABLE | FLORA        | marsilea farinosa             |                                    |
| THREATENED | MAMMALIA     | mellivora capensis            | honey badger                       |
| TRIGGER    | AVES         | mirafra passerina             | monotonous lark                    |
| TRIGGER    | AVES         | monticola rupestris           | cape rock thrush                   |
| TRIGGER    | AVES         | motacilla clara               | mountain wagtail                   |
| TRIGGER    | AVES         | mycteria ibis                 | yellow-billed stork                |
| THREATENED | AVES         | neotis denhami                | denham’s bustard                   |
| VULNERABLE | AVES         | neotis ludwigii               | ludwig’s bustard                   |
| TRIGGER    | AVES         | netta erythrophthalma         | southern pochard                   |
| TRIGGER    | AVES         | nycticorax nycticorax         | black-crowned night heron          |
| ENDANGERED | FLORA        | orbea elegans                 |                                    |
| ENDANGERED | MAMMALIA     | orycteropus afer              | aardvark                           |
| VULNERABLE | FLORA        | oryza longistaminata          |                                    |
| TRIGGER    | REPTILIA     | p. guttatus                   | dwarf flat lizard                  |
| TRIGGER    | REPTILIA     | p. relictus                   | relict flat lizard                 |
| THREATENED | MAMMALIA     | panthera leo                  | lion                               |
| THREATENED | MAMMALIA     | panthera pardus               | leopard                            |
| THREATENED | MAMMALIA     | paracynictis selousi          | selous mongoose                    |
| ENDANGERED | MAMMALIA     | parahyaena brunnea            | brown hyaena                       |
| TRIGGER    | FLORA        | paspalidium germinatum        |                                    |
| TRIGGER    | AVES         | passer motitensis             | great sparrow                      |
| ENDANGERED | AVES         | pelecanus rufescens           | pink - backed pelican              |
| THREATENED | MAMMALIA     | pipistrellus kuhliisubtilis   | kuhl’s bat                         |
| TRIGGER    | AVES         | platalea alba                 | african spoonbill                  |
| TRIGGER    | REPTILIA     | platysaurus minor             | waterberg flat lizard              |
| TRIGGER    | AVES         | plectropterus gambensis       | spur-winged goose                  |
| VULNERABLE | AVES         | podica senegalensis           | african finfoot                    |
| THREATENED | MAMMALIA     | poecilogale albinucha         | african striped weasel             |
| ENDANGERED | AVES         | poicephalus robustus          | cape parrot                        |
| THREATENED | AVES         | polemaetus bellicosus         | martial eagle                      |
| TRIGGER    | AVES         | polyboroides typus            | african harrier-hawk               |
| TRIGGER    | AVES         | porphyrio alleni              | allen’s gallinule                  |
| TRIGGER    | AVES         | porzana pusilla               | baillon’s crake                    |
| TRIGGER    | AVES         | promerops gurneyi             | gurney’s sugarbird                 |
| TRIGGER    | REPTILIA     | prosymna bivittata            | two-striped shovel-snout           |
| TRIGGER    | FLORA        | protea roupelliae             |                                    |
| THREATENED | MAMMALIA     | proteles cristatus            | aardwolf                           |
| VULNERABLE | FLORA        | prunus africana               |                                    |
| TRIGGER    | REPTILIA     | psammobates oculiferus        | kalahari tent tortoise             |
| THREATENED | REPTILIA     | psammophis jallae             | jalla’s sand snake                 |
| TRIGGER    | REPTILIA     | pseudocordylus transvaalensis | northern crag lizard               |
| VULNERABLE | AVES         | pterocles gutturalis          | yellow-throated sandgrouse         |
| ENDANGERED | REPTILIA     | python sebae natalensis       | african rock python                |
| THREATENED | AMPHIBIA     | pyxicephalus adspersus        | giant bullfrog                     |
| TRIGGER    | AMPHIBIA     | pyxicephalus edulis           | african bullfrog                   |
| TRIGGER    | MAMMALIA     | redunca arundinum             | reedbuck                           |
| VULNERABLE | MAMMALIA     | rhinolophus blasii            | peak-saddle horseshoe bat          |
| VULNERABLE | AVES         | rostratula benghalensis       | greater painted-snipe              |
| THREATENED | AVES         | sagittarius serpentarius      | secretarybird                      |
| TRIGGER    | AVES         | sarothrura boehmi             | streaky-breasted flufftail         |
| VULNERABLE | FLORA        | sartidia jucunda              |                                    |
| ENDANGERED | AVES         | scotopelia peli               | pel’s fishing owl                  |
| VULNERABLE | PISCES       | serranochromis meridianus     | lowveld largemouth                 |
| TRIGGER    | AVES         | sporopipes squamifrons        | scaly-feathered finch              |
| THREATENED | MAMMALIA     | suncus linex                  | greater dwarf shrew                |
| THREATENED | MAMMALIA     | syncerus caffer               | african buffalo                    |
| VULNERABLE | AVES         | terathopius ecaudatus         | bateleur                           |
| TRIGGER    | AVES         | thalassornis leuconotus       | white-backed duck                  |
| ENDANGERED | AVES         | torgos tracheliotus           | lappet-faced vulture               |
| VULNERABLE | AVES         | trigonoceps occipitalis       | white-headed vulture               |
| TRIGGER    | AVES         | turdoides bicolor             | southern pied babbler              |
| TRIGGER    | AVES         | turdus libonyanus             | kurrichane thrush                  |
| VULNERABLE | AVES         | tyto capensis                 | african grass owl                  |
| TRIGGER    | AVES         | uraeginthus granatinus        | violet-eared waxbill               |
| TRIGGER    | FLORA        | vachellia                     |                                    |
| TRIGGER    | AVES         | vidua regia                   | shaft-tailed whydah                |
| ENDANGERED | FLORA        | warburgia salutaris           | pepper-bark tree                   |
| TRIGGER    | AVES         | white-bellied sunbird         | cinnyris talatala                  |
| TRIGGER    | REPTILIA     | zygaspis vandami              | van dam’s round-headed worm lizard |
